deployment/digital-ocean/README.md

```diff
@@ -24,3 +24,15 @@ Digital Ocean kubernetes clusters don't have a graphical interface, so I suggest
 to setup the [kubernetes dashboard](./dashboard/README.md) as a next step.
 Configuring [HTTPS](./https/README.md) is bit tricky and therefore I suggest to
 do this as a last step.
+
+## Spaces
+
+We are storing our images in the s3-compatible [DigitalOcean Spaces](https://www.digitalocean.com/docs/spaces/). 
+
+We still want to take backups of our images in case something happens to the images in the cloud. See these [instructions](https://www.digitalocean.com/docs/spaces/resources/s3cmd-usage/) about getting set up with `s3cmd` to take a copy of all images in a `Spaces` namespace, ie `human-connection-uploads`.
+
+After configuring `s3cmd` with your credentials, etc. I was able to make a backup with this command.
```

Comment:
"I"? Hope there is more than one person writing the docs :thinking:

Reply:
that is a good point...